### DISCUSSION TITLE
SEO v2.2 update help

### DISCUSSION THREAD

**Discussion by pensivedog** [gethyas/hyas#466] [Q&A]
Tried updating to v2.2 and getting the following error:
```
execute of template failed: template: partials/seo/opengraph.html:7:15: executing "partials/seo/opengraph.html" at <partial "_funcs/get-page-images" .>: error calling partial: partial "_funcs/get-page-images" not found
```

**Comment by h-enk** [comment]
Cannot replicate it with a fresh install (and adding images) — do you have a repo?

**Reply by h-enk** [reply]
Possibly related: https://github.com/gohugoio/hugo/issues/12210

**Reply by pensivedog** [reply]
Ok, I tried doing a fresh install and noticed its package.json file contains quite a few differences from the one I had after following the Hyas v2 upgrade guide. I added these differences to my package.json and no longer receive this error, but now there is a different error related to images... Will troubleshoot a bit more and start a new thread if I can't figure it out.
